Error building from source

**Version**
Built with 4fdd620

**What happened**

When running `make` I get the following error

```
   • building...
   • loading config file       file=.goreleaser.yml
   • loading environment variables
   • getting and validating git state
      • building...               commit=4fdd620de7f8ae6a749bff970902702bb7c65a0a latest tag=v3.1.1
   ⨯ build failed after 0.21s error=git tag v3.1.1 was not made against commit 4fdd620de7f8ae6a749bff970902702bb7c65a0a
make: *** [Makefile:30: fastly] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -1,3 +1,6 @@
+.PHONY: default
+default: build ;
+
 SHELL := /bin/bash -o pipefail
 
 # the rationale for using both `git describe` and `git rev-parse` is because

